media-gfx/openvdb/openvdb-7.0.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
More Pythons?

Reply:
I think it is OK to keep around other python versions here. Openvdb does not interpret user addons at runtime like blender does. They all support 2.7, 3.5, 3.6, 3.7 and 3.8, but I have only included 3_{6,7,8} in view of the push to retire python 3.6.

Reply:
I'm told we're keen on keeping Python 3.6 for now because of downstreams, but if 3.9 works, can we include that too?

Reply:
I will work more on this tomorrow, and test all versions against python 3.6 to 3.9 and update as appropriate

Reply:
No worries. Thanks for your work. I know it's a lot of effort and a lot of interrelated packages.

Reply:
Based on testing I have added python 3.6 through 3.9 to all versions of openvdb